lib/api/mapBenefits.test.js

```diff
@@ -14,8 +14,18 @@ import SEBMockData from '../../mockdata/userDefault/seb.json'
 import { StatusCodes } from '../../constants/StatusCodes'
 import { SummaryTypes } from '../../constants/SummaryTypes'
 
-describe('maps EI', () => {
-  it('Application Received status', () => {
+const options = { year: 'numeric', month: 'numeric', day: 'numeric' }
+const timezone = 'T12:00:00.000Z'
+
+describe('Maps EI', () => {
+  it('Application Received status - Without estimated decision', () => {
```

Comment:
This doesn't actually check that the estimated decision is NOT returned. Again, check for other places where this applies.